src/Sylius/Bundle/CoreBundle/Form/Type/Product/ChannelPricingType.php

```diff
@@ -35,7 +36,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 $form = $event->getForm();
 
                 $form->add('price', MoneyType::class, [
-                    'label' => $channel->getName(),
+                    'label' => 'Price for ' . $channel->getName(),
```

Comment:
1. What is the reason of this change? I know the UI is not perfect for now, but is it really required? (provide some screenshots if you don't mind 😄 )
2. If we need this, there should be a translation key.